skimage/data/lbpcascade_frontalface_opencv.xml

```diff
@@ -0,0 +1,1505 @@
+<?xml version="1.0"?>

```

Comment:
The name states where this file comes from, but also add a comment into the file itself showing its origin (URL, description, etc.)